liiv2021a/openreview/content.tex

```diff
@@ -0,0 +1,339 @@
+    \item EnD: A single model distilling ENSM trained according to \cite{hinton2015distilling}. 
+    \item \EnDD: A single model distribution-distilling ENSM trained according to \cite{malinin2019ensemble}.   
+    \item EnD$_{\texttt{+AUX}}$: Like EnD, but trained with auxiliary data.
+    \item \EnDDaux: Like \EnDD, but trained with auxiliary data.
+    \item PN$_{\texttt{+AUX}}$: A PriorNet model with auxiliary data trained according to \cite{NIPS2018_7936}
+\end{itemize}
+
+These models are all based on almost identical VGG16 architectures \cite{VGG}, adapted to CIFAR-10 data as in \cite{NIPS2018_7936} by adding dropout, batch normalization and reducing the size of the fully connected layers. The only exception being that batch normalization is not used for PN. 
+
+\subsection{Dataset}
+The training set of CIFAR-10 was used as the primary training dataset. The training set of CIFAR-100 was used as an auxiliary dataset. For evaluating the classification task the test set of CIFAR-10 was used. For evaluating the out-of-distribution detection task the CIFAR-10 test set was used as in-domain dataset, while the LSUN test set was used as the out-of-domain dataset. Information about the datasets is listed in Table \ref{tab:datasets}.
+
+\begin{table}[]
+    \centering
+    \caption{Datasets used in the CIFAR-10 setting}
+    \addtolength{\leftskip} {-3cm}
+    \addtolength{\rightskip}{-3cm}
+    \begin{tabular}{r||c|c|c|c} 
+    \hline
+    Dataset & Samples & Classes & Dimensions & Link \\ [0.5ex] 
+    \hline
+    \hline
+    CIFAR-10 train & 50000 & 10 & 32x32x3 & \hrref{https://www.cs.toronto.edu/~kriz/cifar.html} \\
+    CIFAR-100 train & 50000 & 100 & 32x32x3 & \hrref{https://www.cs.toronto.edu/~kriz/cifar.html} \\
+    CIFAR-10 test & 10000 & 10 & 32x32x3 & \hrref{https://www.cs.toronto.edu/~kriz/cifar.html} \\
+    LSUN test & 10000 & 10 & 256x256x3 & \hrref{https://www.yf.io/p/lsun} \\
+    \hline
+    \end{tabular}
+
+    \label{tab:datasets}
+\end{table}
+
+Each image $x$ was normalized according to $x' = x/127.5 - 1$ where the operations are elementwise, causing all values to lie in the range (-1, 1). The LSUN images were also scaled down to 32x32. Furthermore, dataset augmentation was used for all models, consisting of rotations with 15\degree \ range, horizontal flips, width and height shifts of up to 4 pixels in each direction, and using nearest-neighbour interpolation.
+
+\subsection{Hyperparameters}
+The models were trained with the hyperparameters listed in Table \ref{tab:cifar10-training}. 
+
+\begin{table}
+\centering
+\caption{Training parameters in the CIFAR-10 setting}
+\addtolength{\leftskip} {-3cm}
+\addtolength{\rightskip}{-3cm}
+\begin{tabular}{l||c|c|c|c|c|c|c|c|c}
+\hline
+Model & Epochs & Cycle &  $\eta_0$ & $\eta_{max}$ & $\eta_{min}$ & Dropout & $T_0$ & Anneal & AUX data \\ [0.5ex] 
+\hline
+\hline
+DNN & 45 & 30 & 10^{-3} & 10^{-2} & 10^{-6} & 0.5 & - & - & - \\
```

Comment:
Compilation issue: `! Missing $ inserted`. Please wrap the number `10^{-3}` in `$..$` -> `$10^{-3}`